spec/lib/cylon-i2c.spec.js

```diff
@@ -14,8 +15,8 @@ var BlinkM = lib("blinkm"),
   Pca9544a = lib("pca9544a"),
   Mag3110 = lib("mag3110"),
   Hmc5883l = lib("hmc5883l"),
-  Mma7660 = lib("mma7660"),
-  Mma8452q = lib("mma8452q"),
+//Mma7660 = lib("mma7660"),
```

Comment:
The UnitTest was failing from my branch, those 2 drivers returned a null instead of the expected objects. I've not been able to find why...